lib/util/git/conflicts-cache.ts

```diff
@@ -1,56 +1,42 @@
+import { logger } from '../../logger';
 import { getCache } from '../cache/repository';
 
 export function getCachedConflictResult(
```

Comment:
Why keep this in lib/util/git anymore - can't all this cache logic be centralized into a "branch cache" location? Or do you prefer to do that after?

Reply:
This thought never crossed my mind. It will be good to have them somewhere separate though

Reply:
OK, to be done in a follow-up PR